packages/components/src/Notice/index.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import styled, { StyledComponentClass } from "styled-components";
+import { colors, fonts } from "../styleConstants";
+import { InfoNotification } from "../icons";
+
+export enum NoticeTypes {
+  INFO,
+  ERROR,
+  ALERT,
+}
+
+export interface NoticeContainerProps {
+  type: NoticeTypes;
+}
+
+const typeColors = {
+  [NoticeTypes.INFO]: {
+    border: "rgb(208,237,237)",
+    background: "rgb(208,237,237)",
+  },
+  [NoticeTypes.ERROR]: {
+    border: "rgb(242, 82, 74,1)",
+    background: "rgb(242, 82, 74,0.1)",
+  },
+  [NoticeTypes.ALERT]: {
+    border: "rgba(94,94,94,0.12)",
+    background: "#FFFDE9",
+  },
+};
+
+export const NoticeBackground = styled.div`
+  border-radius: 4px;
+  background-color: #fff;
+`;
+
+export const NoticeContainer = styled<NoticeContainerProps, "div">("div")`
+  display: flex;
+  flex-direction: row;
+  background-position: 10px center;
+  background-size: 30px;
+  background-repeat: no-repeat;
```

Comment:
Maybe I'm missing something but what's the purpose of background position/size/repeat when there's no background image?

Reply:
@tobek Good catch, fixed.